packages/hash/api/src/graphql/typeDefs/paginationConnections.typedef.ts

```diff
@@ -23,6 +23,36 @@ export const pagePaginationTypedef = gql`
     nextPageCursor: String
   }
 
+  type EntityRef {
```

Comment:
I moved these legacy things out of page to persist the pagination typedefs. We don't really need this, and this entire file and its resolver could be deleted as well.

Reply:
Why keep it then?

Reply:
I suppose we could get rid of it, will see how much it breaks with codegen

Reply:
Please do, thanks!

Reply:
Removed in a5f80242a582547c9a976c1b885e37721193731c